packages/manager/modules/web-universe-components/src/exchange/exchange.service.js

```diff
@@ -619,9 +619,10 @@ export default class Exchange {
     );
     delete accountToUpdate.deleteOutlook;
 
-    accountToUpdate.displayName = account.displayName
-      ? account.displayName.trim()
-      : undefined;
+    accountToUpdate.displayName =
+      account.displayName !== (undefined || null)
```

Comment:
![image](https://user-images.githubusercontent.com/22835814/164002286-b60c5d94-54e3-4cb5-9a69-23c7a1060770.png)
Can you check the behaviour if account.displayName is `undefined`?

Reply:
It could be simplified with
```suggestion
      account.displayName != null
```

Reply:
it's applied